PHNX-189 | Доработка Помощницы Рэрити и её окружения

McFord23 opened the pull request.

McFord23 committed:
PHNX-189 | Рвущие носочки

> Обновлена швейня. Теперь у неё прозрачные окна. На маникены повешана сники-броня.
> Добавлено зеркало

McFord23 committed:
PHNX-189 | Обновлён диалог с Помощницей Рэрити

McFord23 committed:
PHNX-189 | Улучшена работа зеркала

Ваза предоставил оригинальные скрипты зеркала из Phoenix 2. Мною они были переведены из GDScript в C#.

McFord23 committed:
PHNX-189 | Добавлена мультизадачность

VaJau2 commented on assets/scripts/triggers/ChangeTaskTrigger.cs:
```diff
@@ -2,20 +2,29 @@
+    
+    private Messages messages;
+    
+    private enum Type
```
Переименуй пж в TaskType

VaJau2 commented on assets/scripts/triggers/ChangeTaskTrigger.cs:
```diff
@@ -29,13 +38,21 @@ public override void SetActive(bool newActive)
-        if (!(body is Player)) return;
-        _on_activate_trigger();
-    }
 }
```
Добавь пж пустую строку в конце

McFord23 committed:
PHNX-197 | Обновлено отображение задач

> Теперь практически все задачи выполняемые.
> Добавлено отображение дополнительных задач.

> Добавлена опция слушать из позиции катсцен-камеры.

PHNX-198 | Фикс анимации бега при усаживании на кресло

McFord23 commented:
Я посчитал эти правки недостойными коммита. Поэтому дополнительно к ним, распространил обновлённое отображение задач на все остальные уровни, зафиксил кресла в Театре, сделал галочку для слушания звуков с позиции камеры (то Винди достал орать лишь в одно ухо) и немножечко оптимизировал зеркало с:
